website/docs/configurations.md

```diff
@@ -1792,6 +1792,16 @@ Configurations controlling the behavior of Kafka source in Hudi Streamer.
 | [hoodie.streamer.source.kafka.value.deserializer.class](#hoodiestreamersourcekafkavaluedeserializerclass)   | io.confluent.kafka.serializers.KafkaAvroDeserializer  | This class is used by kafka client to deserialize the records.<br />`Config Param: KAFKA_AVRO_VALUE_DESERIALIZER_CLASS`<br />`Since Version: 0.9.0`                                                                                                                                                                                                                                                                                                                                                                                                                                         |
 ---
 
+#### Parquet DFS Source Configs {#Parquet-DFS-Source-Configs}
```

Comment:
Config page is automatically generated.  Just to double check, did you use the [tool](https://github.com/apache/hudi/tree/asf-site/hudi-utils) to generate these changes?